Guide.docc/IncrementalAdoption.md

```diff
@@ -149,6 +149,60 @@ class UIStyler {
 Combining static and dynamic isolation can be a powerful tool to keep the
 scope of changes gradual.
 
+## Missing Annotations
+
+Dynamic isolation gives you tools to express isolation at runtime.
+But, you may also find you need to describe other concurrency properties
+that are missing from unmigrated modules.
+
+### Unmarked Sendable Closures
+
+A callback that is missing a `Sendable` annotation can be a problem when
+combined with actor-isolated types.
+
+```swift
+@MainActor
+class PersonalTransportation {
+    func configure() {
+        // This function is missing a Sendable annotation
+        JPKJetPack.jetPackConfiguration {
+            // MainActor isolation will be inferred here, incorrectly
+            self.applyConfiguration()
+        }
+    }
+
+    func applyConfiguration() {
+    }
+}
+```
+
+If `jetPackConfiguration` can invoke its closure in another isolation domain,
+it must be marked `@Sendable`.
+When an un-migrated module hasn't yet done this, it will result in incorrect
+actor inference.
+The above code, which does contain dynamic isolation verification checks,
```

Comment:
```suggestion
The above code would compile without issue, however it could crash at runtime,
if the closure were to be invoked from a different isolation domain than the target
actor's, because under Swift 6 runtime checks are inserted at such language boundaries.
```

(Is this the right "reason" for why there is a check injected here? @xedin @hborla)

Reply:
I would love this clarification. But, above that, I do also want to point out that as of latest betas, this does not acually crash for me: https://github.com/apple/swift-migration-guide/issues/63

Reply:
Hmmmm, in Swift 6 mode rigth? I'm surprised... let's double check that, thank you

Reply:
Let's track that in #63 , which I will not close until we're sure we understand this.